WalletWasabi.Gui/Global.cs

```diff
@@ -300,6 +302,273 @@ public static async Task InitializeNoWalletAsync()
 			Logger.LogInfo("Start synchronizing filters...");
 		}
 
+		private static async Task AddKnownBitcoinFullNodeAsHiddenServiceAsync(AddressManager addressManager)
+		{
+			#region Onion seed list 
+			//  curl -s https://bitnodes.21.co/api/v1/snapshots/latest/ | egrep -o '[a-z0-9]{16}\.onion:?[0-9]*' | sort -ru
```

Comment:
Can you translate this to C#? So I can add to the packager and refresh it at every release?

Reply:
Also don't forget the testnet!

Reply:
I didn't find any api/site providing a similar list for testnet. However, using the dns seed I can see that there are only two tor addresses registered and they are probably not available because wasabi cannot connect to those.

Here you have the code for generating the list:
https://gist.github.com/lontivero/3fcc9cbb145a427fbae305ece873c516#file-program-cs

Here you have the code for querying the dns seed and get the list of tor addresses:
https://gist.github.com/lontivero/3fcc9cbb145a427fbae305ece873c516#file-program2-cs